app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -55,14 +55,19 @@ keywords:
   - Settlement Scheme
 ---
 
+
+[International relocation payment applications for teachers and salaried trainees who started their job or course in September 2023 are now open](https://get-a-teacher-relocation-payment.education.gov.uk/). You must apply by 31 October 2023.  
+
+[If you are a trainee on a fee-paying course, you do not need to apply.](#how-trainee-teachers-can-get-the-international-relocation-payment)
```

Comment:
I'm not sure it's clear where the link for this takes you. Perhaps it needs a specific 'Learn more' link?

Reply:
I think you've done this :)